integration-tests/src/test/java/oracle/kubernetes/operator/SitConfig.java

```diff
@@ -76,11 +76,13 @@ protected  Domain createSitConfigDomain(boolean domainInImage, String domainScri
    *
    * @throws Exception when domain destruction fails
    */
-  protected static void destroySitConfigDomain(Domain domain) throws Exception {
+  protected static void destroySitConfigDomain(Domain domain, boolean domainInImage) throws Exception {
```

Comment:
Actually, I'm not sure hiding this here helps.  The caller has the domain object and could just call domain.deleteImage().

Reply:
same goes for domain.destroy() also and don't even need this method destroySitConfigDomain(). I think the method was added here as there are two sit config test classes, so that they both inherit these methods.

Reply:
destroySitConfigDomain() not really doing much, I can put those two lines in the test class itself.